avocado/utils/dmesg.py

```diff
@@ -115,3 +122,19 @@ def collect_errors_by_level(output_file=None, level_check=5):
             err += " Please check  dmesg log in debug log."
             LOGGER.debug(dmsg_log)
         raise DmesgError("Test is failed {}".format(err))
+
+
+def skip_dmesg_msg(dmesg_messages, skip_dmesg):
+    """Remove dmesg messages which want to skip.
+
+    :dmesg_messages: dmesg messages from which want to filter
```

Comment:
dmesg_messages give us the idea of "list of things". Give us the idea of a "list"/plural. Since this is a pure stdout buffer, my suggestion is to rename this to "dmesg_buffer" or "dmesg_stdout".